mitodl/micromasters
# incorporate homepage template

#### What are the relevant tickets?

this accomplishes part of #97 

#### What's this PR do?

implements a temporary version of the homepage

#### Where should the reviewer start?

the stuff going on in `cms/templates/cms/home_page.html` and `/static/scss`.

#### How should this be manually tested?

Pull down the branch, check it out, click around and make sure I didn't clobber the CSS for any other part of the side.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

![mm_homepage](https://cloud.githubusercontent.com/assets/6207644/14713109/ca458cf8-07ad-11e6-87c7-f2bc66aacb5c.png)

#### What GIF best describes this PR or how it makes you feel?
